app/Http/Controllers/SetupController.php

```diff
@@ -51,8 +51,18 @@ public static function displaySetupPage(Request $request) {
         if (env('POLR_SETUP_RAN')) {
             return self::setupAlreadyRan();
         }
-
-        return view('setup');
+        
+        // see if any of the variables have already been provided in .env file
+        // in which case pass those to the view, otherwise assume defaults
+        $view_data = array(
```

Comment:
We should  use new style array syntax: ` $view_data = [ ... ]`

Reply:
Cool, I can change this easily enough. 

I never took to the new shorthand way of creating arrays because it looked more like JSON and I sometimes have to type out JSON arrays when working in JS files and its nice to keep things distinct. Are there some advantages other than visual appeal that I am unaware of? If so I'll begin using it more in my own projects.